[0m[[0minfo[0m] [0mRunning ru.sayn.simple_gui.AdminGui [0m
[0m[[0mdebug[0m] [0mWaiting for threads to exit or System.exit to be called.[0m
[0m[[0mdebug[0m] [0m  Classpath:[0m
[0m[[0mdebug[0m] [0m	C:\workspace\scala\SimpleGUI\target\scala-2.10\classes[0m
[0m[[0mdebug[0m] [0m	C:\Users\root\.sbt\boot\scala-2.10.3\lib\scala-library.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\root\.ivy2\cache\org.scala-lang\scala-actors\jars\scala-actors-2.10.3.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\root\.ivy2\cache\postgresql\postgresql\jars\postgresql-9.1-901-1.jdbc4.jar[0m
[0m[[0mdebug[0m] [0m	C:\Users\root\.ivy2\cache\com.typesafe\config\bundles\config-1.2.1.jar[0m
[0m[[0mdebug[0m] [0mWaiting for thread run-main-2 to terminate.[0m
[0m[[31merror[0m] [0m(run-main-2) java.lang.ClassNotFoundException: ru.sayn.simple_gui.AdminGui[0m
java.lang.ClassNotFoundException: ru.sayn.simple_gui.AdminGui
	at java.lang.ClassLoader.findClass(ClassLoader.java:358)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
	at sbt.classpath.ClasspathFilter.loadClass(ClassLoaders.scala:53)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:249)
	at sbt.Run.getMainMethod(Run.scala:77)
	at sbt.Run.run0(Run.scala:64)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:54)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:58)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:58)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:58)
	at sbt.Logger$$anon$4.apply(Logger.scala:90)
	at sbt.TrapExit$App.run(TrapExit.scala:244)
	at java.lang.Thread.run(Thread.java:662)
[0m[[0mdebug[0m] [0m	Thread run-main-2 exited.[0m
[0m[[0mdebug[0m] [0mInterrupting remaining threads (should be all daemons).[0m
[0m[[0mdebug[0m] [0mSandboxed run complete..[0m
java.lang.RuntimeException: Nonzero exit code: 1
	at scala.sys.package$.error(package.scala:27)
	at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:1666)
	at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:1666)
	at scala.Option.foreach(Option.scala:236)
	at sbt.BuildCommon$class.toError(Defaults.scala:1666)
	at sbt.Defaults$.toError(Defaults.scala:35)
	at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$35$$anonfun$apply$36.apply(Defaults.scala:651)
	at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$35$$anonfun$apply$36.apply(Defaults.scala:649)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42)
	at sbt.std.Transform$$anon$4.work(System.scala:64)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18)
	at sbt.Execute.work(Execute.scala:244)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:662)
[0m[[31merror[0m] [0m(compile:[31mrunMain[0m) Nonzero exit code: 1[0m
